.github/test/scripts/check_notebook_output.py

```diff
@@ -77,6 +77,7 @@
     "Detected ",
     "FutureWarning: promote has been superseded by mode",
     "dataframe_reader.complete_incoming_dataframe",
+    "Method connections: This is an experimental method",
```

Comment:
The warning is from ml_client.workspaces.get, which is not experimental.
It also only happens from the PR [SDK Samples Run generated from 3766778 by azure-sdk · Pull Request #3166 · Azure/azureml-examples (github.com)](https://github.com/Azure/azureml-examples/pull/3166)
I think that PR should be fixed to not produce the message